CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+## TBD
+
+### Bug fixes
+
+* Synchronize access to sessions being delivered in de-duplication code
```

Comment:
would be good to include a link to the PR here

Reply:
Apologies - I had most of these in an unpushed commit!